polls/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from accounts.models import CustomUser
+
+# Create your models here.
+
+class Company(models.Model):
+    name = models.CharField(max_length=100)
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+class Poll(models.Model):
+    company = models.ForeignKey(Company, related_name='polls', on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    price = models.IntegerField()
+    category = models.ForeignKey(Category, on_delete=models.DO_NOTHING)
+    short_description = models.TextField(max_length=50)
+    description = models.TextField(max_length=500)
+    time_needed = models.TimeField()
```

Comment:
I think this field stores not certain time (e.g hour) , but time period, so it's better to use DurationField here. Exaggerated example would be poll that has `time_needed` bigger than 24 hours, this would be not possible to represent with TimeField, but possible with DurationField

https://docs.djangoproject.com/en/3.1/ref/models/fields/#durationfield